mail/rspamd/Portfile

```diff
@@ -25,34 +24,34 @@ long_description    ${description}  Rspamd core is written in C language using e
 checksums           rmd160  a8dd52a6ede4c52289b23a69d3ea80602e46696e \
                     sha256  dd1d8e9cdbb1c870be3c1bfee57a6894b6f8174595bc0684212a7c1afa4439f0 \
                     size    4460935
-                 
-compiler.blacklist-append   *gcc* {clang < 800}
-compiler.whitelist  clang macports-clang-8.0 macports-clang-7.0 macports-clang-6.0 macports-clang-5.0
+ 
+compiler.blacklist-append *gcc* {clang < 800}
+compiler.whitelist clang macports-clang-8.0 macports-clang-7.0 macports-clang-6.0 macports-clang-5.0
 
 depends_build-append \
                     port:pkgconfig
 
-depends_lib-append  \
-                    port:fann \
-                    port:gd2 \
-                    path:lib/pkgconfig/glib-2.0.pc:glib2 \
-                    port:gmime \
-                    port:libevent \
+depends_lib-append  port:libevent \
                     port:libmagic \
+                    port:sqlite3 \
+                    port:openssl \
+                    port:ragel \
+                    port:luajit \
                     port:libstemmer \
+                    port:fann \
+                    port:gd2 \
                     port:libunwind \
+                    port:hyperscan \
+                    port:gmime \
                     port:lua \
-                    port:luajit \
-                    path:lib/libopenblas.dylib:OpenBLAS \
-                    port:openssl \
                     port:perl5 \
-                    port:ragel \
-                    port:sqlite3 \
-                    port:zlib
+                    port:zlib \
+                    path:lib/pkgconfig/glib-2.0.pc:glib2 \
+                    path:lib/libopenblas.dylib:OpenBLAS
```

Comment:
Please sort these dependencies.

Reply:
Is this latest edit what you mean?

Reply:
Just alphabetically sort them. For the path ones, I manually put them in the right place afterwards.

Reply:
I believe the issue is the changes here where not based on top of the latest version in master. The simplest fix is to start afresh from that version, make the minimal changes needed, and force push that.

Reply:
Ok thanks, didn't see I was out-of-sync. Think it's fixed now…